spec/dummy/app/models/person.rb

```diff
@@ -5,5 +5,9 @@ class Person
   validates_presence_of :name
 
   attr_accessor :ni_number
+  attr_accessor :email_work
+  attr_accessor :email_home
```

Comment:
Looks like we're not using the `email_home` field in the specs. Shall we delete it?